brave-lists/debounce.json

```diff
@@ -39,7 +39,9 @@
       "*://*.mailchimp.com/mctx/clicks?*",
       "*://*.tradedoubler.com/click?*",
       "*://cc.techbargains.com/v1/otc/*",
-      "*://www.anrdoezrs.net/click-*"
+      "*://www.anrdoezrs.net/click-*",
```

Comment:
If you insert the new ones at the beginning of the list instead of the bottom, then it helps preserves the `git blame` by removing these unnecessary comma changes.

Reply:
we can just use `git log -Sanrdoezrs.net`  to show the commit

Reply:
Sure, there's alternatives, it's not the end of the world. I was just pointing out that it's just as easy to add things to the top and then this view works better: https://github.com/brave/adblock-lists/blame/master/brave-lists/debounce.json